src/app/pages/TopicPage/HierarchicalGrid/index.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { string, shape, arrayOf, number, oneOfType, oneOf } from 'prop-types';
+import {
+  GEL_GROUP_1_SCREEN_WIDTH_MAX,
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_2_SCREEN_WIDTH_MAX,
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '#psammead/gel-foundations/src/breakpoints';
+import Promo, { MEDIA_TYPES } from '#components/Promo';
+import { DESKTOP, TABLET, MOBILE, SMALL } from './dataStructures';
+
+const PromoList = styled.ul`
+  padding: 1.5rem;
+  margin: 0;
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: repeat(4, 1fr);
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    grid-template-columns: repeat(3, 1fr);
+  }
+  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+    grid-template-columns: repeat(2, 1fr);
+  }
+`;
+
+const Item = styled.div`
+  position: relative;
+  display: inline;
+  background-color: white;
+  padding: 1rem;
+`;
+
+const getStyles = (promoCount, i) => {
+  return [
+    css`
+      @media (max-width: ${GEL_GROUP_1_SCREEN_WIDTH_MAX}) {
+        ${SMALL[promoCount - 1][i]}
+      }
+      @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
+        ${MOBILE[promoCount - 1][i]}
+      }
+      @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+        ${TABLET[promoCount - 1][i]}
+      }
+      @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+        ${DESKTOP[promoCount - 1][i]}
+      }
+    `,
+  ];
+};
+
+const HiearchicalGrid = ({ summaries }) => {
+  if (!summaries || summaries.length < 3) return null;
+  const summaryItems = summaries.slice(0, 12);
+  return (
+    <PromoList role="list">
+      {summaryItems.map((promo, i) => {
+        return (
+          <Item key={promo.id} css={getStyles(summaryItems.length, i)} as="li">
+            <Promo>
+              <Promo.Image
+                src={promo.imageUrl}
+                alt={promo.imageAlt}
+                loading="lazy"
```

Comment:
Could the `promo-image` classname be applied here rather than in the promo itelf: https://github.com/bbc/simorgh/pull/10313/files#diff-5348c3cc0ae1af4758ae84e7d22cf211507ea980a6f40d952b423b9563f89accR45?

It feels more cohesive to keep the classnames needed for the styles applied here to be in the same file so it's clearer how they work together? 

I'd say the same for `promo-text` but I'm unsure how you would do that at this level? 
https://github.com/bbc/simorgh/pull/10313/files#diff-5348c3cc0ae1af4758ae84e7d22cf211507ea980a6f40d952b423b9563f89accR46